modules/aggregation-layers/src/utils/cpu-aggregator.ts

```diff
@@ -316,9 +321,9 @@ export default class CPUAggregator {
     });
   }
 
-  getDimensionChanges(oldProps, props, changeFlags) {
+  getDimensionChanges(oldProps: any, props: any, changeFlags: any): Function[] | null {
```

Comment:
ditto. 

Cant props be `Record<string, unknown>`?

Can't updater be a bit more specific, Function is very loose.

Reply:
I'll be honest this class is a total mess and I would rather spend time rewrite it than trying to type it.